pulp_python/app/__init__.py

```diff
@@ -8,4 +10,4 @@ class PulpPythonPluginAppConfig(PulpPluginAppConfig):
 
     name = "pulp_python.app"
     label = "python"
-    version = "3.2.0.dev"
+    version = __version__
```

Comment:
I am unsure if this will confuse our release automation, which handles version bumps for us.  https://github.com/pulp/pulp_python/blob/master/.bumpversion.cfg#L17

Potentially not, but we would need to test it manually.  Is there a functional reason for this change or primarily stylistic?